src/isIsogram.test.js

```diff
@@ -6,4 +6,27 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true 
+    if word is an isogram`, () => {
+    expect(isIsogram(''))
+      .toBeTruthy();
+  });
+
+  it(`should return true 
```

Comment:
You should add a test case for non-empty isogram words as well. For example, you can test with the word 'isogram' and expect it to be true.